package.json

```diff
@@ -79,6 +79,7 @@
     "eslint-config-airbnb-base": "^14.0.0",
     "eslint-config-prettier": "^6.7.0",
     "eslint-plugin-import": "^2.19.1",
+    "fs-extra": "^10.0.0",
```

Comment:
seems not used in this changes, lets remove it.

Reply:
`fs-extra` is a peer dependency of fixbind. By default it uses `fs.rmSync`, but for older versions of node, like the one used in Atom, it falls back to `fse.removeSync`.

I can add a @note in package.json if needed.